homeassistant/components/sensor/waqi.py

```diff
@@ -77,7 +77,7 @@ def name(self):
         try:
             return self._details['city']['name']
```

Comment:
Got you. Cuz I saw a red AQI label under the circle, thought it is already very obvious.
<img width="451" alt="screen shot 2017-01-10 at 2 36 22 pm" src="https://cloud.githubusercontent.com/assets/1425206/21796244/5b3a5982-d742-11e6-8ad5-2ddc9b50e92a.png">
Below is a screenshot when I first added only one city into configuration. It shows up way more info then I needed.
<img width="1052" alt="screen shot 2017-01-10 at 2 44 34 pm" src="https://cloud.githubusercontent.com/assets/1425206/21796445/cc8c4702-d743-11e6-9ce1-cdbfad7855af.png">

But you are right that all other sensors have a prefix of their sensor brand as prefix (like Yweathter). Maybe a global config to turn on/off all the prefix is a better solution. I would change it back then.